subresource-integrity/integrity-policy/tentative/signature.https.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/dispatcher/dispatcher.js"></script>
+<script src="/common/utils.js"></script>
+<script src="/reporting/resources/report-helper.js"></script>
+
+<body>
+<script>
+  promise_test(async () => {
+    // Generated using https://sha2.it/ed25519.html (In Chrome Canary, with Experimental Web Platform Features enabled)
```

Comment:
Maybe I don't understand enough, but does this need to be open in chrome canary? I thought you just paste what you need. If so, specifying which version might be more useful than Chrome canary :)

Reply:
The hash needs to be generated in a browser that supports the feature. I'll change the comment.